src/main/java/com/psipher/application/actions/GeneratePasswordOperations.java

```diff
@@ -0,0 +1,16 @@
+package com.psipher.application.actions;
+
+import com.psipher.application.exceptions.DDBException;
+
+/**
+ * Functions to generate random passwords.
+ */
+
+public interface GeneratePasswordOperations {
+    /**
+     * It generates a random password for users.
+     * @param length Length of password
+     * @return password
+     */
+    String generatePassword(Integer length) throws DDBException;
```

Comment:
You dont need to throw this exception it is not relevant

Reply:
I got your point, I have made changes. Please check.